lib/lambda_ethereum_consensus/telemetry.ex

```diff
@@ -142,6 +142,18 @@ defmodule LambdaEthereumConsensus.Telemetry do
       ),
       last_value("fork_choice.fetch.exception.duration",
         unit: {:native, :millisecond}
+      ),
+      last_value("subnet.persist.stop.duration",
```

Comment:
Not important for this PR, but I think there's a way in which we can have only one metric for this and then have metadata saying for which table this is. In this way we can unify this for all table stores/gets and then filter in grafana instead.

Reply:
I'll create an issue for this!